evalml/tests/automl_tests/test_sequential_engine.py

```diff
@@ -13,6 +18,53 @@ def test_evaluate_no_data():
         engine.evaluate_batch([])
 
 
+def test_add_ensemble_data():
```

Comment:
@bchen1116 I like your division of test coverage here! The first test checks the accessors, and the second checks the functionality using mocking. Very nice.